plugins/badges-backend/src/service/router.ts

```diff
@@ -33,6 +33,7 @@ export interface RouterOptions {
   catalog?: CatalogApi;
   config: Config;
   discovery: PluginEndpointDiscovery;
+  env: any;
```

Comment:
let's pass `tokenManager` instead so that we can have all the typings in place 😄

```suggestion
  tokenManager: TokenManager;
```